src/views/assets/Platform/PlatformList.vue

```diff
@@ -115,6 +115,11 @@ export default {
                 vm.$route.query._type = row.type.value
                 vm.$route.query._category = row.category.value
                 vm.$refs.genericListTable.onUpdate({ row, col })
+              },
+              onClone: ({ row, col }) => {
+                vm.$route.query._type = row.type.value
+                vm.$route.query._category = row.category.value
+                vm.$refs.genericListTable.onClone({ row, col })
               }
             }
           }
```

Comment:
There doesn't appear to be anything specific in this snippet of code that needs addressing at the moment since it is quite simple.

However, if you want some general improvements like adding comments explaining the purpose of the function/method or ensuring consistent style (like spacing and indentation), we'd need more details about which area you wish to analyze.